src/mrunal/prog1.java

```diff
@@ -0,0 +1,42 @@
+package mrunal;
+/* assignemnt 18 -----Program1 ------
+ * : max repeating word with its freq.
+	input : Hi Hello Hi Techno Hello Hi
+	output : Hi -> 3 */
+
+public class prog1 
+{
+ static void displayCount(String str) 
+ {
+			String [] strArray=str.split(" ");
+			int max = 0;
+			String maxName = null; 
+			for (int index = 0; index < strArray.length;index++)
+			{
+				int count = 0;
+				String word = strArray[index];	
+				if (str.contains(word)) 
```

Comment:
Hi Mrunal, here you pick word from array n check whether that is in main string or not, this word definitely exist in that main string right?
So here contains condition not make sense much